src/client/components/board/board.jsx

```diff
@@ -98,6 +98,8 @@ class Board extends React.Component {
       this.props.ctx.activePlayers[this.props.playerID] === 'threats'
         ? true
         : false;
+    // If this.props.playerID is unset, then we are in spectator mode and check whether the first player is in threat stage.
```

Comment:
maybe we could introduce some descriptive functions instead of the comment. Something like:
```jsx

const isSpectator = () => !!this.props.playerID;
const isFirstPlayerInThreatStage = () => this.props.ctx.activePlayers?.[0] === 'threats';

const shouldShowTimer = isInThreatStage || (isSpectator() && isFirstPlayerInThreatStage());
```